docs/content/users/quickstart.md

```diff
@@ -397,6 +397,9 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
     ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true
     ```
 
+!!!tip "Add Vite support?"
+    Since Laravel v9.19, Vite is included as the default [asset bundler](https://laravel.com/docs/master/vite). There are small tweaks needed in order to use it: [Working with Vite in DDEV - Laravel](https://ddev.com/blog/working-with-vite-in-ddev/#laravel).
```

Comment:
```suggestion
    Since Laravel v9.19, Vite is included as the default [asset bundler](https://laravel.com/docs/vite). There are small tweaks needed in order to use it: [Working with Vite in DDEV - Laravel](https://ddev.com/blog/working-with-vite-in-ddev/#laravel).
```

Reply:
Thanks for the change, but let's use this link https://laravel.com/docs/vite, which will always redirects to the latest stable docs, because master means unstable.

Reply:
Sorry, I moved too fast, trying to clean up.